### PR TITLE
Add support for subscript, superscript, code annotations.

### PR DESCRIPTION
**Pull request by michael**

### CODE REVIEW

File: packages/code/CodeCommand.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+var AnnotationCommand = require('../../ui/AnnotationCommand');
+var oo = require('../../util/oo');
+
+function CodeCommand() {
+  AnnotationCommand.apply(this, arguments);
+}
+oo.inherit(CodeCommand, AnnotationCommand);
+
+CodeCommand.static = {
```
Comment (obuchtala) [inline packages/code/CodeCommand.js]: This is may cause problems as you loose the prototype chain of the static object. I.e. you loose static inheritance.

File: packages/subscript/Subscript.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var PropertyAnnotation = require('../../model/PropertyAnnotation');
+
+var Subscript = PropertyAnnotation.extend({
+  name: 'subscript'
```
Comment (obuchtala) [inline packages/subscript/Subscript.js]: When we stick with Class.extend, we should remove any custom magic.
I.e. this should go into a static block.

File: packages/subscript/SubscriptCommand.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+var AnnotationCommand = require('../../ui/AnnotationCommand');
+var oo = require('../../util/oo');
+
+function SubscriptCommand() {
+  AnnotationCommand.apply(this, arguments);
+}
+oo.inherit(SubscriptCommand, AnnotationCommand);
+
+SubscriptCommand.static = {
```
Comment (obuchtala) [inline packages/subscript/SubscriptCommand.js]: This must not be an object.

File: packages/subscript/SubscriptTool.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+var AnnotationTool = require('../../ui/AnnotationTool');
+var oo = require('../../util/oo');
+
+function SubscriptTool() {
+  AnnotationTool.apply(this, arguments);
+}
+
+oo.inherit(SubscriptTool, AnnotationTool);
+
+SubscriptTool.static = {
```
Comment (obuchtala) [inline packages/subscript/SubscriptTool.js:12]: DITO

File: packages/superscript/SuperscriptCommand.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+var AnnotationCommand = require('../../ui/AnnotationCommand');
+var oo = require('../../util/oo');
+
+function SuperscriptCommand() {
+  AnnotationCommand.apply(this, arguments);
+}
+oo.inherit(SuperscriptCommand, AnnotationCommand);
+
+SuperscriptCommand.static = {
```
Comment (obuchtala) [inline packages/superscript/SuperscriptCommand.js]: DITO

File: packages/superscript/SuperscriptTool.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+var AnnotationTool = require('../../ui/AnnotationTool');
+var oo = require('../../util/oo');
+
+function SuperscriptTool() {
+  AnnotationTool.apply(this, arguments);
+}
+
+oo.inherit(SuperscriptTool, AnnotationTool);
+
+SuperscriptTool.static = {
```
Comment (obuchtala) [inline packages/superscript/SuperscriptTool.js:12]: DITO

**Comment by michael** [comment]
Saw it from your other posts. I' away from computer for few hours. Will fix later

**Comment by obuchtala** [comment]
Needs fixing.

**Comment by michael** [comment]
Thanks for comments!

**Comment by obuchtala** [comment]
Fixed.

**Pull request merged**